### DISCUSSION TITLE
Update for blog

### DISCUSSION THREAD

**Discussion by MitanshuShaBa** [timlrx/tailwind-nextjs-starter-blog#334] [Q&A] [answered]
Once I have used this template and customized it according to me, what is the best way to update my codebase with the new features that are added in this template?

**Comment by timlrx** [comment, marked as answer]
If you are already happy with it, I don't see a need to upgrade. For smaller bug fixes you can just copy the changes. Or you can just upgrade the dependencies if required. Some of those have breaking changes so you can consult the changelog for a release upgrade to the changes required.

For larger changes, you can [pull the latest version](https://tailwind-nextjs-starter-blog.vercel.app/blog/new-features-in-v1#upgrade-guide) and work through the changes. In general, you should have some understanding of js / react / nextjs to make the necessary amendments.

**Comment by eerison** [comment]
Hey @MitanshuShaBa 

If you have customized many files, I would say you gonna get many conflicts, but it isn't impossible to update! 

I just wrote 2 articles with 2 different alternatives. I hope it can help you.

https://blog.erison.work/posts/get-frontend-theme-updates-using-git-part-1/

https://blog.erison.work/posts/get-frontend-theme-updates-using-git-part-2/
